.github/workflows/build-git-installers.yml

```diff
@@ -204,4 +204,94 @@ jobs:
         with:
           name: win-${{matrix.artifact.name}}-x86_64
           path: artifacts
-  # End build Windows installers
\ No newline at end of file
+  # End build Windows installers
+
+  # Build Mac OSX installers & upload artifacts
+  mac_artifacts:
+    runs-on: macos-latest
+    needs: prereqs
+    env:
+      # `gettext` is keg-only
+      LDFLAGS: -L/usr/local/opt/gettext/lib
+      CFLAGS: -I/usr/local/opt/gettext/include
+      # Link with cURL
+      CURL_LDFLAGS: -lcurl
+      # To make use of the catalogs...
+      XML_CATALOG_FILES: /usr/local/etc/xml/catalog
+      # Enable a bit stricter compile flags
+      DEVELOPER: 1
+      # For the osx-installer build
+      OSX_VERSION: 10.6
```

Comment:
Put this on the list of something we _could_ update in our fork of `git_osx_installer`, but this is a good workaround for now.